api/app/Models/Pool.php

```diff
@@ -87,7 +87,8 @@ public function nonessentialSkills(): BelongsToMany
     /* accessor to obtain Advertisement Status, depends on two variables regarding published and expiry */
     public function getAdvertisementStatusAttribute()
     {
-        date_default_timezone_set('America/Vancouver');
+        // given database is functioning in UTC, all backend should consistently enforce the same timezone
+        date_default_timezone_set('UTC');
```

Comment:
Do we need this line at all? I believe laravel automatically sets the timezone to UTC for us.

Reply:
aye aye